README.md

```diff
@@ -12,13 +12,15 @@ This terraform module is used to integrate AWS into a meshStack instance as a me
 
 - [Terraform installed](https://learn.hashicorp.com/tutorials/terraform/install-cli)
 - [AWS CLI installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+- Three accounts to segregate accounts by function.
+  - management account: organization management account, the account that hosts the AWS Organization
+  - meshcloud account: meshStack will use this account to host the IAM users used by meshStack
```

Comment:
s: we should probably change the naming of this to "meshStack" account to avoid any confusion potential here

Reply:
Agreed, I opened a separate PR for that: https://github.com/meshcloud/terraform-aws-meshplatform/pull/17